aqt/metadata.py

```diff
@@ -551,28 +554,29 @@ def __init__(
                 self.request_type = "modules"
                 version, arch = modules_query
                 self._action = lambda: self.fetch_modules(self._to_version(version, arch), arch)
-        elif architectures_ver:
+        elif architectures_ver is not None:
+            ver_str: str = architectures_ver
             self.request_type = "architectures"
-            self._action = lambda: self.fetch_arches(self._to_version(architectures_ver, None))
+            self._action = lambda: self.fetch_arches(self._to_version(ver_str, None))
```

Comment:
`ver_str` should be unnecessary, but mypy complains about using `architectures_ver` here